next.config.js

```diff
@@ -9,4 +9,5 @@ module.exports = {
   },
 
   target: 'serverless',
+  swcMinify: true,
```

Comment:
If i'm not wrong, as long as we have a `.babelrc` file, Next will disable SWC, so this line probably doesn't do much lul